BookingSystem/Frontend/src/shared/settings/links.js

```diff
@@ -0,0 +1,6 @@
+const links = {
+  HOTEL_SEARCH: "/hotel/",
+  HOTEL_PAGE: id => `hotel/${id}/rooms`
```

Comment:
it should be a constant according to the Uppercase naming convention, but not a function. Think about different approach or naming